### PR TITLE
♻️ added: JSON Beautify 

### PR DESCRIPTION
**Pull request by priyaranjan12345**

### CODE REVIEW

**Comment by priyaranjan12345** [comment]
api_dash_beautify
api_dash_beautify1

File: lib/widgets/request_widgets.dart [hunk 1]
```diff
@@ -46,13 +46,14 @@ class _RequestPaneState extends State<RequestPane>
...
     }
     return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
       children: [
         Padding(
           padding: kPh20v10,
           child: SizedBox(
             height: kHeaderHeight,
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
```
Comment (ashitaprasad) [inline lib/widgets/request_widgets.dart:55]: Why did you modify Row into Wrap?

File: lib/widgets/request_widgets.dart [hunk 2]
```diff
@@ -65,10 +66,9 @@ class _RequestPaneState extends State<RequestPane>
                         ? Icons.code_off_rounded
                         : Icons.code_rounded,
                   ),
-                  label: SizedBox(
```
Comment (ashitaprasad) [inline lib/widgets/request_widgets.dart:68]: Again. What is the rationale behind this change.

File: lib/widgets/response_widgets.dart [hunk 3]
```diff
@@ -407,22 +408,31 @@ class _BodySuccessState extends State<BodySuccess> {
         return Padding(
           padding: kP10,
           child: Column(
+            crossAxisAlignment: CrossAxisAlignment.stretch,
```
Comment (ashitaprasad) [inline lib/widgets/response_widgets.dart:411]: Why these changes till line 451?

File: lib/widgets/raw.dart [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import 'dart:convert';
+
+import 'package:apidash/providers/providers.dart';
+import 'package:apidash/widgets/snackbars.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class Raw extends StatelessWidget {
```
Comment (ashitaprasad) [inline lib/widgets/raw.dart:8]: Raw can be any type of data. Currently, you are assuming that it is JSON.

**Comment by ashitaprasad** [comment]
The purpose of `Raw` is to show the raw response. But, some users will find this feature useful so in stead of adding an obstructing button, it can be provided as a switch in the setting where the user can select whether they want to see formatted response or the raw unformatted response for JSON response.

File: lib/widgets/request_widgets.dart (hunk 1 above)
Comment (priyaranjan12345) [reply lib/widgets/request_widgets.dart:55]: due to overflow issue I changed row to wrap. Also I have changed `preview` and `raw` button to short button on small screen size
screen overflow

Comment (ashitaprasad) [reply lib/widgets/request_widgets.dart:55]: We already have min width requirement in place which is enforced in release version, so it is not required.
